packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
```

Comment:
No need to clone `children`, because the fixture is never mutated. If we went down this road we'd have to clone props and other stuff as well. Let's just send `fixture.children` directly to `React.createElement`.